tests/test_billboard.py

```diff
@@ -0,0 +1,413 @@
+            "59c73209de645ef7e4e4293f4d6882b3",
+        ],
+        "lab_majmin": [
+            "annotations/0003/majmin.lab",
+            "59c73209de645ef7e4e4293f4d6882b3",
+        ],
+        "lab_majmininv": [
+            "annotations/0003/majmininv.lab",
+            "59c73209de645ef7e4e4293f4d6882b3",
+        ],
+    }
+
+    # assert track.audio_path == 'tests/resources/mir_datasets/McGill-Billboard/' + 'audio/1960s/James Brown/I Don\'t Mind/audio.flac'
+    assert track.title == "I Don't Mind"
+    assert track.artist == "James Brown"
+
+    # # test that cached properties don't fail and have the expected type
+    assert type(track.sections) is utils.SectionData
+
+    # # test audio loading functions
+    # y, sr = track.audio
+    # assert sr == 44100
+    # assert y.shape == (89856,)
+
+
+def test_to_jams():
+    data_home = "tests/resources/mir_datasets/McGill-Billboard"
+    track = billboard.Track("3", data_home=data_home)
+    jam = track.to_jams()
+
+    segments = jam.search(namespace="segment")[0]["data"]
+    print([segment.time for segment in segments])
+    assert [segment.time for segment in segments] == [
+        0.073469387,
+        22.346394557,
+        49.23802721,
+        76.123990929,
+        102.924353741,
+        130.206598639,
+    ]
+
+    assert [segment.duration for segment in segments] == [
+        22.27292517,
+        26.891632653,
+        26.885963719000003,
+        26.800362812000003,
+        27.282244897999988,
+        20.70278911600002,
+    ]
+
+    assert [segment.value for segment in segments] == [
+        ("A", "intro"),
+        ("B", "verse"),
+        ("B", "verse"),
+        ("A", "interlude"),
+        ("B", "verse"),
+        ("A", "interlude"),
+    ]
+
+    assert [segment.confidence for segment in segments] == [
+        None,
+        None,
+        None,
+        None,
+        None,
+        None,
+    ]
+
+    assert jam["file_metadata"]["title"] == "I Don't Mind"
+    assert jam["file_metadata"]["artist"] == "James Brown"
+
+
+def test_track_ids():
+    track_ids = billboard.track_ids()
+    assert type(track_ids) is list
+    assert len(track_ids) == 890
+
+
+def test_load():
+    data_home = "tests/resources/mir_datasets/McGill-Billboard"
+    data = billboard.load(data_home=data_home)
+    assert type(data) is dict
+    assert len(data.keys()) == 890
+
+    # data home default
+    data_default = billboard.load()
+    assert type(data_default) is dict
+    assert len(data_default.keys()) == 890
+
+
+def test_load_chords():
+    # load a file which exists
+    chords_path = (
+        "tests/resources/mir_datasets/McGill-Billboard/" + "annotations/0035/full.lab"
+    )
+    chords_data = billboard._load_chords(chords_path)
+
+    # check types
+    assert type(chords_data) == utils.ChordData
+    assert type(chords_data.intervals) is np.ndarray
+    assert type(chords_data.labels) is list
+
+    # check values
+    assert chords_data.labels == [
+        "N",
```

Comment:
What about removing all but a few chords from the label file for the tests? Then this test can be much smaller.